tuftool/src/remove_key_role.rs

```diff
@@ -0,0 +1,124 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT OR Apache-2.0
+
+use crate::datetime::parse_datetime;
+use crate::error::{self, Result};
+use crate::source::parse_key_source;
+use chrono::{DateTime, Utc};
+use snafu::ResultExt;
+use std::fs::File;
+use std::num::NonZeroU64;
+use std::path::PathBuf;
+use structopt::StructOpt;
+use tempfile::tempdir;
+use tough::editor::targets::TargetsEditor;
+use tough::http::HttpTransport;
+use tough::key_source::KeySource;
+use tough::schema::decoded::{Decoded, Hex};
+use tough::Transport;
+use tough::{ExpirationEnforcement, FilesystemTransport, Limits, Repository};
+use url::Url;
+
+#[derive(Debug, StructOpt)]
+pub(crate) struct RemoveKeyArgs {
+    /// Key files to sign with
+    #[structopt(short = "k", long = "key", required = true, parse(try_from_str = parse_key_source))]
+    keys: Vec<Box<dyn KeySource>>,
+
+    /// Keys to be removes
```

Comment:
:%s/removes/removed

Also, could we add a hint for the user so they know what the argument should be?